### ISSUE TITLE
PlayerLogger doesn't support mysql-connector 8.x

### ISSUE THREAD

**Issue by GhostNaix** [ForgeEssentials/ForgeEssentials#2572]
I'm currently running version ` forgeessentials-1.7.10-1.4.6.1364-server` which is the latest version on the jenkins server
When I try to configure PlayerLogger to log to MySQL for an external tool the server crashes with the following error:

```
[03:22:27] [Server thread/ERROR] [FML/]: Caught exception from ForgeEssentials
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
```
I've install the MySQL Connector/J for Java still no change
Is MySQL no longer supported ?
Is this incorrect configuration ?

**Here are the log files**

[modlist.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/5612459/modlist.txt)
[fml-server-latest.log](https://github.com/ForgeEssentials/ForgeEssentials/files/5612460/fml-server-latest.log)

**Comment by spacebuilder2020** [comment]
MySQL has not been touched in a long time.  The MySQL provider was never recommended and I have a hunch it was never finished.  I will spend some cycles investigating it later.

**Comment by spacebuilder2020** [comment]
At this point, I am assuming you have moved on from FE.   I am still providing an update so others who run into this issue on 1.7.10 can resolve their issue.

However, my initial investigation have revealed this to be a mysql connector 8 error.  A PR a while back for 1.7.10 updated the mysql connector to 8.x but it appears that playerlogger is not compatable with 8.x.  This is only in 1.7.10....  1.12 is still on 5.1.22.    I recommend going to `./ForgeEssentials/lib` and replacing the 8.x jar with the 5.1.22 jar.  You should be able to find it inside of the libaries.zip file in the Forge Essentials 1.12 server jar file.

A later task will be endeavored to update playerlogger to 8.x.  I think this will require updating hibernate so it will be a bit involved.

**Comment by GhostNaix** [comment]
Yeah sorry, I have moved on from Forge Essentials long ago so I cannot provide any further Information or testing.
Apologies for the inconvenience and lack of response

**Comment by spacebuilder2020** [comment]
No worries.  I am guessing that is the issue cause it seems that it was updated to 8.x on 1.7.10 but not 1.12
